### PR TITLE
HAAR-860: Send User enable email using gov notification client

### PR DESCRIPTION
**Pull request by ravmoj**
Send User enable email using gov notification client

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/EmailNotificationControllerIntTest.kt [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        .exchange()
+        .expectStatus().isForbidden
+    }
+
+    @Test
+    fun sendEnableEmail() {
+      externalUsersApiMockServer.stubPostSendEnableEmail()
+      webTestClient.post().uri("/notify/enable-user")
+        .headers(setAuthorisation(roles = listOf("ROLE_MAINTAIN_OAUTH_USERS")))
+        .body(
```
Comment (karenmillermoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/EmailNotificationControllerIntTest.kt:63]: Should this also have AUTH_GROUP_MANAGER?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/EmailNotificationService.kt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    with(emailNotificationDto) {
+      val parameters = mapOf(
+        "firstName" to firstName,
+        "username" to username,
+        "signinUrl" to authBaseUri,
+      )
+      // send the email
+      try {
+        log.info("Sending enable user email to notify for user {}", username)
+        notificationClient.sendEmail(enableUserTemplateId, email, parameters, null)
```
Comment (karenmillermoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/EmailNotificationService.kt:29]: When we do send the ExternalUserEnabledEmailSuccess - or is this done in external users?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/EmailNotificationControllerIntTest.kt (hunk 1 above)
Comment (ravmoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/EmailNotificationControllerIntTest.kt]: added

File: src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/EmailNotificationService.kt (hunk 2 above)
Comment (ravmoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/EmailNotificationService.kt:29]: after enabling user in external users api

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**